src/components/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../styles/Footer.css';
+
+const Footer = () => (
+  <div className="footer">
+    <p>
+      {' '}
+      Designed and coded by:
+      {' '}
+      <a href="https://github.com/Mesi21">@Mesi21</a>
+    </p>
+  </div>
+);
+
+export default Footer;
```

Comment:
- [Optional] The footer of your website is hiding some buttons as you can see in the screenshot below. Kindly consider rectifying the issue.
![annotely_image (4)](https://user-images.githubusercontent.com/63087491/165576346-a16a2cc8-5d79-47d9-81fa-e6a2a6591fb3.png)

Thanks :pray: 
.